package.json

```diff
@@ -32,6 +32,18 @@
 		"url": "https://github.com/anteprimorac/vscode-html-end-tag-labels.git"
 	},
 	"main": "./out/extension.js",
+    "contributes": {
+		"configuration": {
+		    "title": "Html End Tag Labels",
+		    "properties": {
+			"htmlEndTagLabels.labelColor": {
+			    "type": "string",
+			    "default": "#ffffff",
```

Comment:
@ctf0 should the option by default be an empty string?

Reply:
np